pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -256,13 +261,13 @@ type TaskRunStatusFields struct {
 	// +listType=atomic
 	RetriesStatus []TaskRunStatus `json:"retriesStatus,omitempty"`
 
-	// Results from Resources built during the taskRun. currently includes
+	// Results from Resources built during the TaskRun. currently includes
 	// the digest of build container images
 	// +optional
 	// +listType=atomic
 	ResourcesResult []PipelineResourceResult `json:"resourcesResult,omitempty"`
 
-	// TaskRunResults are the list of results written out by the task's containers
+	// TaskRunResults are the list of results written out by the Task's containers
```

Comment:
there's a bunch of these cleanup for unrelated code - can that please go in a separate commit and PR?

Reply:
Thanks Jerop, updated.